src/core_plugins/metrics/public/components/lib/create_text_handler.js

```diff
@@ -1,10 +1,14 @@
 import _ from 'lodash';
+import detectIE from './is-ie';
```

Comment:
can you change this to work like 
```
import { isIE } from './is_ie';
```
or
```
import { detectIE } from './detect_id';
```
We stopped relying on exporting defaults and we also use snake case for all our filenames.